docs/operator/config.yaml

```diff
@@ -143,6 +143,9 @@ featureGates:
     # If empty the operator automatically derives the domain from the cluster.
     baseDomain: ""
 
+    # OAuthProxyEnabled is used internally for enable by default the oauth proxy for the UI in single tenant mode
+    oAuthProxyEnabled: false
```

Comment:
do we ever need to add more config options related to oauth proxy? if so, I'd make it a block (struct) like
```
oauthproxy:
  enabled: false
```